src/components/sectionList/filters/DynamicFilters.tsx

```diff
@@ -7,24 +7,24 @@ import {
     CategoryOptionFilter,
     CategoryOptionGroupFilter,
     DataDimensionTypeFilter,
-    DataSetFilter,
     IndicatorFilter,
     DomainTypeSelectionFilter,
     IgnoreApprovalFilter,
     PublicAccessFilter,
     ValueTypeSelectionFilter,
+    FormTypeFilter,
 } from './filterSelectors'
 import { useFilterKeys } from './useFilterKeys'
 
 type FilterKeyToComponentMap = Partial<Record<ConfigurableFilterKey, React.FC>>
 
 const filterKeyToComponentMap: FilterKeyToComponentMap = {
     category: Categoryfilter,
+    formType: FormTypeFilter,
     indicatorType: IndicatorFilter,
     categoryOption: CategoryOptionFilter,
     categoryCombo: CategoryComboFilter,
     categoryOptionGroup: CategoryOptionGroupFilter,
-    dataSet: DataSetFilter,
```

Comment:
Why is this removed? This is used in the `dataElement`-list.

Reply:
Oh, I actually thought I was the one who added it. will add it back.